.github/.workflows/release-please.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          release-type: node
+          package-name: release-please-action
```

Comment:
I _think_ `package-name` should be `unchained-bitcoin`. I also think you need `command: manifest` here, so that the action looks at the `release-please-config.json` for deets.

Reply:
amended with those changes. thanks